### PR TITLE
Add ability to change max rps/endpoint and capacity scaler

### PR DESCRIPTION
**Pull request by bowei**
Add ability to change max rps/endpoint and capacity scaler

* networking.gke.io/max-rate-per-endpoint
* networking.gke.io/capacity-scaler

### CODE REVIEW

**Comment by bowei** [comment]
/assign @freehan @rramkumar1 

cc: @mark-church

File: pkg/controller/translator/translator.go [hunk 1]
```diff
@@ -128,6 +128,28 @@ func setAppProtocol(sp *utils.ServicePort, svc *api_v1.Service, port *api_v1.Ser
 	return nil
 }
 
+func setTrafficPolicy(sp *utils.ServicePort, svc *api_v1.Service) error {
+	const maxRatePerEndpointKey = "networking.gke.io/max-rate-per-endpoint"
+	if s, ok := svc.Annotations[maxRatePerEndpointKey]; ok {
+		val, err := strconv.ParseFloat(s, 64)
```
Comment (rramkumar1) [inline pkg/controller/translator/translator.go:134]: I think you need to check for empty string before calling ParseFloat

File: pkg/flags/flags.go [hunk 2]
```diff
@@ -236,6 +237,8 @@ L7 load balancing. CSV values accepted. Example: -node-port-ranges=80,8080,400-5
 	flag.StringVar(&F.GKEClusterName, "gke-cluster-name", "", "The name of the GKE cluster this Ingress Controller will be interacting with")
 	flag.StringVar(&F.GKEClusterHash, "gke-cluster-hash", "", "The cluster hash of the GKE cluster this Ingress Controller will be interacting with")
 	flag.StringVar(&F.GKEClusterType, "gke-cluster-type", "ZONAL", "The cluster type of the GKE cluster this Ingress Controller will be interacting with")
+
+	flag.BoolVar(&F.EnableTrafficPolicy, "enable-traffic-policy", false, "Enable support for Service {max-rate-per-endpoint, capacity-scaler}")
```
[the remark this answers is not in the document]
Comment (freehan) [reply pkg/flags/flags.go:241]: IIUC, auto-scaler will also read this config. So it is better to align with that.

**Review by freehan — APPROVED** [review]
/lgtm
/approve

do we want to cherrypick into 1.13?

**Comment by bowei** [comment]
Hey -- don't we need to incorporate the review comments before approve?

File: pkg/flags/flags.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (bowei) [reply pkg/flags/flags.go:241]: How about `EnableTrafficScaling`?

[the remark this answers is not in the document]
Comment (bowei) [reply pkg/flags/flags.go:241]: How about `EnableTrafficScaling`?

[the remark this answers is not in the document]
Comment (rramkumar1) [reply pkg/flags/flags.go:241]: SGTM

**Pull request merged**